apps/accounts/forms.py

```diff
@@ -72,23 +72,6 @@ def __init__(self, *args, **kwargs):
 
 class PasswordResetRequestForm(forms.Form):
     email = forms.EmailField(max_length=255, label=_('Email'))
-    # human_x = randint(1, 9)
```

Comment:
Looks like there's a dup of this in `apps/accounts/views/forms.py` any idea which one's used?

Reply:
It looks like this is the one that's used, judging from the imports, but I honestly haven't spent that much time in this part of the codebase.

Reply:
OK can you clean up the otherone? If there's concern of it being truly dead code or not you could have it inherit from this class so it's just a pass through.